modules/source-formatter.properties

```diff
@@ -12,7 +12,7 @@
         com.fasterxml.jackson-dataformat:jackson-dataformat-cbor:2.12.6,\
         com.fasterxml.jackson.core:jackson-annotations:2.13.2,\
         com.fasterxml.jackson.core:jackson-core:2.13.2,\
-        com.fasterxml.jackson.core:jackson-databind:2.13.2.2,\
+        com.fasterxml.jackson.core:jackson-databind:2.13.4.2,\
```

Comment:
@hongvo2308 Please split the commit into 2, modify source-formatter.properties first, and then commit the auto-SFed changes.